lib/text/ui_text_displayer.js

```diff
@@ -617,6 +617,13 @@ shaka.text.UITextDisplayer = class {
     // TODO: Implement line alignment with line number.
     // TODO: Implement lineAlignment of 'CENTER'.
     if (cue.line != null) {
+      // HACK: the current implementation of UITextDisplayer only handled
+      // PERCENTAGE, as cue.line=0 is zero, we can interpret it as percentage
+      // safely.
+      if (cue.line == 0 &&
+          cue.lineInterpretation == Cue.lineInterpretation.LINE_NUMBER) {
+        cue.lineInterpretation = Cue.lineInterpretation.PERCENTAGE;
```

Comment:
It's not technically listed in the interface or anything, but up until now the process of appending cues has not caused the actual cue objects to be mutated.
I think it'd be better to make a local variable for `lineInterpretation` and change that, here.

Reply:
Changed!